### PR TITLE
Fixes #825: Added Metadata API description to docs

### LINKED ISSUE #825: Document the Metadata API
Dapr has a `v1.0/metadata` API that returns, among other things, a list of activated actors.

This API endpoint is currently undocumented.

### PR DESCRIPTION
**Pull request by juazasan**
## Description

Added Metadata API description to docs.

## Issue reference

This PR will close: #825

### CODE REVIEW

File: daprdocs/content/en/reference/api/metadata_api.md [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+type: docs
+title: "Metadata API reference"
+linkTitle: "Metadata API"
+description: "Detailed documentation on the Metadata API"
+weight: 800
+---
+
+Dapr has a metadata API that returns information about the sidecar allowing runtime discoverability. The metadata endpoint returns among other things, a list of the components loaded and the activated actors (if present).
+
+The Dapr metadata API also allows you to store additional information in they format of key-value pairs.
```
Comment (artursouza) [inline daprdocs/content/en/reference/api/metadata_api.md:11]: ... in the format ...

File: daprdocs/content/en/reference/api/metadata_api.md [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+linkTitle: "Metadata API"
+description: "Detailed documentation on the Metadata API"
+weight: 800
+---
+
+Dapr has a metadata API that returns information about the sidecar allowing runtime discoverability. The metadata endpoint returns among other things, a list of the components loaded and the activated actors (if present).
+
+The Dapr metadata API also allows you to store additional information in they format of key-value pairs. 
+
+Note: The Dapr metatada endpoint is for instance being used by the dapr cli when running dapr in standalone mode to store the PID of the process hosting the sidecar and the command used to run the application.
```
Comment (artursouza) [inline daprdocs/content/en/reference/api/metadata_api.md:13]: Dapr CLI

File: daprdocs/content/en/reference/api/metadata_api.md [hunk 3]
```diff
@@ -0,0 +1,191 @@
...
+
+### HTTP Response Body
+
+**Metadata API Response Object**
+
+Name                   | Type                                                                  | Description
+----                   | ----                                                                  | -----------
+id                     | string                                                                | Application ID
+actors                 | [Metadata API Response Registered Actor](#metadataapiresponseactor)[] | A json encoded array of Registered Actors metadata.
+extended.attributeName | string                                                                | List of custom attributes as key(attributeName)-value pairs.
```
Comment (artursouza) [inline daprdocs/content/en/reference/api/metadata_api.md:46]: ... key-value pairs, where key is the attribute name.

File: daprdocs/content/en/reference/api/metadata_api.md (hunk 1 above)
Comment (juazasan) [reply daprdocs/content/en/reference/api/metadata_api.md]: Thanks for the review. I have just updated the PR to address your comments.

File: daprdocs/content/en/reference/api/metadata_api.md (hunk 3 above)
Comment (juazasan) [reply daprdocs/content/en/reference/api/metadata_api.md]: Thanks for the review. I have just updated the PR to address your comments.

File: daprdocs/content/en/reference/api/metadata_api.md (hunk 2 above)
Comment (juazasan) [reply daprdocs/content/en/reference/api/metadata_api.md]: Thanks for the review. I have just updated the PR to address your comments.

**Review by artursouza — APPROVED** [review]

**Pull request merged**